doc/esmvalcore/recipe.rst

```diff
@@ -80,24 +97,28 @@ For example, a datasets section could be:
 
 
 Note that this section is not required, as datasets can also be provided in the
-`Diagnostics`_ section.
+Diagnostics_ section.
 
+.. _Preprocessors:
 
-Preprocessors
-=============
+Recipe section: ``preprocessors``
+=================================
 
 The preprocessor section of the recipe includes one or more preprocesors, each
-of which may call one or several preprocessor functions.
+of which may call the execution of one or several preprocessor functions.
 
 Each preprocessor section includes:
 
-- A preprocessor name.
-- A list of preprocesor functions to apply
-- Any Arguments given to the preprocessor functions.
-- The order that the preprocesor functions are applied can also be specified using the ``custom_order`` preprocesor function.
+- A preprocessor name (any name, under ``preprocessors``);
+- A list of preprocesor steps to be executed (choose from the API);
+- Any or none arguments given to the preprocessor steps;
+- The order that the preprocesor steps are applied can also be specified using
+  the ``custom_order`` preprocesor function.
 
-The following preprocessor is an example of a preprocessor that contains
-multiple preprocessor functions:
+The following snippet is an example of a preprocessor named ``prep_map`` that
+contains multiple preprocessing steps (:ref:`Horizontal regridding` with two
+arguments, :ref:`Time operations` with no arguments and :ref:`Multi-model
```

Comment:
Why would you call "time_average" with no argument? What will happen then? Nothing? If nothing, why call it?

Reply:
Added an explanation.